src/components/molecules/NavBar/NavBar.scss

```diff
@@ -231,20 +220,81 @@ $photobooth-button-background-size: 35px;
   }
 
   &-clicked {
-    margin-left: $sidebar-schedule-width;
-    transition-property: margin-left;
-    transition: 0.15s;
-    width: $nav-schedule-btn-width-clicked;
-    background: url(/assets/icons/nav-schedule-whatson-clicked.svg)
-      $saturated-black-alt;
-
     &:hover {
       background-color: $lighter-intermediate-grey;
       box-shadow: none;
     }
   }
 }
 
+.nav-clickable {
```

Comment:
Good opportunity to move styles inside the `.NavBar` block

Reply:
fixed in 2bd2a15